regression/smv/modules/self1.desc

```diff
@@ -1,5 +1,5 @@
 KNOWNBUG
-process1.smv
+self1.smv
 
 ^EXIT=0$
```

Comment:
The missing blank line was the root cause of the test spuriously passing in CI. Fixed.